bioptim/dynamics/configure_problem.py

```diff
@@ -312,6 +316,68 @@ def configure_contact_function(ocp, nlp, dyn_func: Callable, **extra_params):
             legend=all_contact_names,
         )
 
+    @staticmethod
+    def configure_soft_contact_function(ocp, nlp):
+        """
+        Configure the soft contact sphere
+
+        Parameters
+        ----------
+        ocp: OptimalControlProgram
+            A reference to the ocp
+        nlp: NonLinearProgram
+            A reference to the phase
+        """
+        nlp.soft_contact_forces_func = []
+        for i_sc in range(nlp.model.nbSoftContacts()):
+            soft_contact = nlp.model.softContact(i_sc)
+            n = int(nlp.states.mx_reduced.shape[0] / 2)
+            softcontact_force_func = (
+                biorbd.SoftContactSphere(soft_contact)
+                .computeForceAtOrigin(nlp.model, nlp.states.mx_reduced[:n], nlp.states.mx_reduced[n:])
+                .to_mx()
+            )
+
+            nlp.soft_contact_forces_func.append(
+                Function(
+                    "soft_contact_forces_func",
+                    [nlp.states.mx_reduced, nlp.controls.mx_reduced, nlp.parameters.mx],
+                    [softcontact_force_func],
+                    ["x", "u", "p"],
+                    ["soft_contact_forces"],
+                ).expand()
+            )
+
+            all_soft_contact_names = []
+            l = ["Mx", "My", "Mz", "Fx", "Fy", "Fz"]
```

Comment:
Why `l`?

Reply:
I don't know, i needed these, Here.

Reply:
I just needed it.

Reply:
No I meant, what does mean `l`? Why this name?